src/coreComponents/mesh/PerforationData.cpp

```diff
@@ -128,9 +128,11 @@ void PerforationData::computeWellTransmissibility( MeshLevel const & mesh,
       WellElementRegion const & wellRegion = dynamicCast< WellElementRegion const & >( wellElemSubRegion.getParent().getParent() );
       GEOS_UNUSED_VAR( wellRegion ); // unused if geos_error_if is nulld
       GEOS_LOG_RANK_IF( isZero( m_wellTransmissibility[iperf] ),
-                        "\n \nWarning! A perforation is defined with a zero transmissibility in " << wellRegion.getWellGeneratorName() << "! \n" <<
+                        "\n \nWarning! Perforation " << wellRegion.getWellGeneratorName() <<
+                        " is defined with a zero transmissibility.\n" <<
                         "The simulation is going to proceed with this zero transmissibility,\n" <<
-                        "but a better strategy to shut down a perforation is to remove the <Perforation> block from the XML\n \n" );
+                        "but a better strategy to shut down a perforation is to remove the " <<
+                        "<Perforation> block from the XML\n \n" );
```

Comment:
Why did you modify this file?

Reply:
Because I thought GEOS code should be 100 character long.

At first, I started to try to access the `InternalWell::getDataContext()` here, but with no success. I have a question about that [here](https://github.com/GEOS-DEV/GEOS/pull/2450#discussion_r1327367396).